tests/_pure_get_prompt_symbol.test.fish

```diff
@@ -1,14 +1,14 @@
-source $current_dirname/fixtures/constants.fish
-source $current_dirname/../functions/_pure_get_prompt_symbol.fish
-@mesg (_print_filename $current_filename)
+source (dirname (status filename))/fixtures/constants.fish
+source (dirname (status filename))/../functions/_pure_get_prompt_symbol.fish
+@echo (_print_filename (status filename))
 
 
 function setup
     _purge_configs
     _disable_colors
     set --universal pure_symbol_prompt '❯'
     set --universal pure_symbol_reverse_prompt '❮'
-end
+end; setup
```

Comment:
If you are not using `setup` anywhere else, you might as well remove the `setup` function and just leave the code.

Reply:
I prefer to keep the function, it helps explicit what the code does

Reply:
I'd agree if we were using the function more than once in the same file, but we are not.